doc/source/install/index.rst

```diff
@@ -40,8 +40,9 @@ without further installation or configuration.
 So effectively, using a package manager like pip it is sufficient to
 install the packages like this:
 
-.. code-block: console
-   $ pip install openstackclcient otcextensions
+.. code-block::
```

Comment:
Is it correct that you removed console syntax highlighting?

Reply:
it is not relevant there